README.md

```diff
@@ -267,6 +268,65 @@ mobiledoc-specific props:
   interface or not.
 - `isEditing`: A bool indicating if the card is in Edit mode or not.
 
+## Component-based Atoms
+
+As stated in the [Mobiledoc Atom Documentation](https://github.com/bustle/mobiledoc-kit/blob/master/ATOMS.md), "Atoms are effectively read-only inline cards." They are sections of rich content that only spans the space of a word or a sentence within a paragraph. The common example is an `@` mention within a block of text.
+
+`react-mobiledoc-editor` comes with a helper for using your own React
+components as the display and update the content of an Atom.
+
+To wrap your own component in the Atom interface, simply call `classToDOMAtom`
+on it:
+
+```jsx
+import { Component } from 'react';
+import { classToDOMAtom } from 'react-mobiledoc-editor';
+
+class MyComponent extends React.Component<Props> {
+  static displayName = 'MyComponent';
+
+  handleClick = () => {
+    let {payload, save, value} = this.props;
+    let clicks = (payload.clicks || 0) + 1;
+    save(value, {...payload, clicks});
+  };
+
+  render() {
+    let { payload } = this.props;
+
+    return (
+      <button onClick={this.handleClick}>
+        Clicks: {payload.clicks || 0}
+      </button>
+    );
+  }
+}
+
+const MyComponentAtom = classToDOMAtom(MyComponent);
+```
```

Comment:
@joshfrench I recycled the "[Example dom atom that uses the save hook](https://github.com/bustle/mobiledoc-kit/blob/master/ATOMS.md#atom-examples)
" from the atom docs for demonstration purposes here. It's not as simple as merely displaying the payload content. But I found that example most personally helpful to get my head around how to work with atoms.

Let me know if you'd rather have a simpler example for the README though.

Reply:
🤔 Maybe add this atom to the demo and use something like a simple `@` mention in the readme?

Reply:
ok @joshfrench I simplified this example, and added ClickCounterAtom to the demo